src/v2/Apps/Partner/Components/Overview/NearbyGalleriesRail.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+import { Box, BoxProps, Flex, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { NearbyGalleriesRail_partners } from "v2/__generated__/NearbyGalleriesRail_partners.graphql"
+import { NearbyGalleriesRailRendererQuery } from "v2/__generated__/NearbyGalleriesRailRendererQuery.graphql"
+import { Carousel } from "../Carousel"
+import { useSystemContext } from "v2/Artsy"
+import { NearbyGalleryCardFragmentContainer } from "./NearbyGalleryCard"
+import { compact } from "lodash"
+import { NearbyGalleriesRailPlaceholder } from "./NearbyGalleriesRailPlaceholder"
+
+interface NearbyGalleriesRailProps extends BoxProps {
+  partners: NearbyGalleriesRail_partners
+  city: string | null
+}
+
+const NearbyGalleriesRail: React.FC<NearbyGalleriesRailProps> = ({
+  partners,
+  city,
+  ...rest
+}) => {
+  if (!partners || partners.length === 0) {
+    return null
+  }
+
+  return (
+    <Box {...rest}>
+      <Flex mb={4} justifyContent="space-between" alignItems="center">
+        <Text variant="title">Nearby Galleries</Text>
+      </Flex>
+
+      <Carousel itemsPerViewport={[2, 2, 3]}>
+        {partners.map(({ node }) => {
+          return (
+            <NearbyGalleryCardFragmentContainer
+              key={node?.id}
```

Comment:
If `node` can be null here, can we just remove those cases from the `partners` array before iterating through?

Reply:
Did you mean something like this?
![image](https://user-images.githubusercontent.com/79979820/118988238-aed84800-b989-11eb-9be6-d89b986d8841.png)

In this case, we still have the same error
![image](https://user-images.githubusercontent.com/79979820/118988286-bc8dcd80-b989-11eb-8d27-e2e1cfed1649.png)

Reply:
Yeah that's what I meant. Surprised we're still getting that-- @dzucconi (or others), is this expected?

Reply:
Filter doesn't necessarily return the correct type I believe. Just `if (!node) return null` should be fine though.

Reply:
`if (!node) return null` also results in a type error because `Carousel` doesn't work with empty children.
![image](https://user-images.githubusercontent.com/79979820/119334107-64a9dc00-bc93-11eb-879a-b02980968965.png)

I used `compact(partners.map(c => c.node))`. Let me know if you are okay with it.